lib/parser/openmp-grammar.h

```diff
@@ -0,0 +1,356 @@
+// Copyright (c) 2018, NVIDIA CORPORATION.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef OPENMP_PARSER_GRAMMAR_H_
+#define OPENMP_PARSER_GRAMMAR_H_
+
+// Top-level grammar specification for Fortran.  These parsers drive
```

Comment:
This looks like an unedited cut-and-paste from grammar.h.

Reply:
I just fixed that in my branch.